eth/filters/bor_filter_test.go

```diff
@@ -0,0 +1,158 @@
+package filters
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	types "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/params"
+
+	gomock "github.com/golang/mock/gomock"
+)
+
+var (
+	key1, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
+	addr    = crypto.PubkeyToAddress(key1.PublicKey)
+)
+
+func newTestHeader(blockNumber uint) *types.Header {
+	return &types.Header{
+		Number: big.NewInt(int64(blockNumber)),
+	}
+}
+
+func newTestReceipt(contractAddr common.Address, topicAddress common.Hash) *types.Receipt {
+	receipt := types.NewReceipt(nil, false, 0)
+	receipt.Logs = []*types.Log{
+		{
+			Address: contractAddr,
+			Topics:  []common.Hash{topicAddress},
+		},
+	}
+
+	receipt.Bloom = types.CreateBloom(types.Receipts{receipt})
+
+	return receipt
+}
+
+func (backend *MockBackend) expectBorReceiptsFromMock(hashes []*common.Hash) {
+	for _, h := range hashes {
```

Comment:
I believe something like that already was somewhere in the comments. Could you remove code duplication?

Also is it possible to review all gomock.Any() usage and use it as least as possible?

Reply:
https://github.com/maticnetwork/bor/pull/456#discussion_r924367178

https://github.com/maticnetwork/bor/pull/456#discussion_r924368597

This might answer this.

Reply:
For gmock.Any() : https://github.com/maticnetwork/bor/pull/456#discussion_r924383586

Reply:
Also, https://github.com/maticnetwork/bor/pull/456#discussion_r924378572